cli/src/templates/packages/nativewind/components/EditScreenInfo.tsx.ejs

```diff
@@ -0,0 +1,40 @@
+import { Text, View } from 'react-native';
+
+<% if (props.internalizationPackage?.name === "i18next") { %>
+import { useTranslation } from 'react-i18next';
+<% } %>
+
+
+export const EditScreenInfo = ({ path }: { path: string }) => {
+<% if (props.internalizationPackage?.name === "i18next") { %>
+  const { t } = useTranslation();
+  const getStarted = t('getStarted');
+  const changeCode = t('changeCode')
+<% } else { %>
+  const getStarted = "Open up the code for this screen:"
```

Comment:
change these to "title" and "description", something more generic

Reply:
done !